tests/common/macro_support.hpp

```diff
@@ -10,7 +10,7 @@
  * @file Contains support macros for the testcase helper macros. These macros are implementation details, and thus
  * should not be used directly. Use their frontends instead.
  */
-
+#if 0
```

Comment:
same here, can we just delete this file now?   nothing shoudl be using the old Macros anymore